app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :trackable, :lockable, :timeoutable and :omniauthable
+  extend Devise::Models
```

Comment:
Is this needed? I thin the DTA concern have all the stuff you need

Reply:
Without this line, the app not recognize device gem functions. I searched in stackoverflow and this solution work for me. I'll try again.

Reply:
It's ok, that a valid reason